dev/docker/docker-compose.yml

```diff
@@ -0,0 +1,38 @@
+version: '3'
+services:
+  chaos:
+    image: chaos
+    build:
+      context: ../..
+      dockerfile: ./dev/docker/Dockerfile
+    command: bash /root/workspace/Chaos/dev/docker/start_services.sh
+    links:
```

Comment:
Yes you're right! An old habit… I will fix it!
(`db` is not used for now but it will serve in future)